libqtile/backend/x11/core.py

```diff
@@ -577,6 +577,8 @@ def handle_ButtonPress(self, event) -> None:  # noqa: N802
         self.qtile.process_button_click(
             button_code, state, event.event_x, event.event_y, event
         )
+        self.conn.conn.core.AllowEvents(xcffib.xproto.Allow.ReplayPointer, event.time)
+        self.conn.conn.flush()
```

Comment:
Overall this looks good to me, though I wonder what this flush is for. Seems like it should work without this?

Reply:
(I understand that you copy/pasted it, I'm wonder if we can just drop it)

Reply:
Probably useless, yeah.